src/main/java/org/colonelkai/guielements/nodes/settings/SettingBox.java

```diff
@@ -0,0 +1,82 @@
+package org.colonelkai.guielements.nodes.settings;
+
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.control.Labeled;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.layout.HBox;
+import javafx.scene.text.Font;
+import org.colonelkai.ForwardLauncher;
+import org.colonelkai.application.Settings;
+
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Type;
+
+public class SettingBox<T> extends HBox {
```

Comment:
Is this a settings box? As in what the user enters a value into?

Reply:
No It's an HBox containing 2 other components, one being the label of the setting and other being the interactable (a switch or smh)

Reply:
Ok so its a label and a dynamic input field (depending upon the input, so a switch would be boolean for example)?

Reply:
Yes